lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1134,7 +1136,15 @@ toApiNetworkParameters (NetworkParameters gp sp pp) txConstraints toEpochInfo =
             $ view #decentralizationLevel pp
         , desiredPoolNumber = view #desiredNumberOfStakePools pp
         , minimumUtxoValue = toApiCoin $
-            txOutputMinimumAdaQuantity txConstraints TokenMap.empty
+            -- NOTE: It's difficult for us to return anything good here. We
+            -- should really - at least additionally - return the raw
+            -- lovelacePerUTxOWord or lovelacePerUTxOByte parameters.
+            --
+            -- This is now returning the minUTxOValue for the longest shelley
+            -- address, 9-byte coin, and no tokens.
+            txOutputMinimumAdaQuantity txConstraints
+                (AD.maxLengthAddress $ Proxy @ShelleyKey)
+                TokenMap.empty
```

Comment:
Unfortunate. Would like to see `Cardano.ProtocolParameters` used for the JSON instead ([related](https://input-output.atlassian.net/browse/ADP-2046?atlOrigin=eyJpIjoiZDc3NTVlZWZkNjllNGQwMzhmZDIwYTQ1NmZhYjk4MTkiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)). Could also use `longestAddressOfAnyType`, but I think some integration test broke then.

Reply:
> Unfortunate. Would like to see `Cardano.ProtocolParameters` used for the JSON instead ([related](https://input-output.atlassian.net/browse/ADP-2046?atlOrigin=eyJpIjoiZDc3NTVlZWZkNjllNGQwMzhmZDIwYTQ1NmZhYjk4MTkiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)). Could also use `longestAddressOfAnyType`, but I think some integration test broke then.

I agree with you, this is very unfortunate. I also agree that we should deprecate this field and use `Cardano.ProtocolParameters` instead.

In the meantime, I've written an explanation for users in this commit: 3a5f5cf0974b52126bfa0c333c4c93006267708a